website/docs/r/workflow.html.markdown

```diff
@@ -331,9 +331,12 @@ resource "newrelic_workflow" "workflow-example" {
 }
 ```
 
-## Additional Information
+## Moving from Legacy Alert Policy Channels to Workflows
 More details about the workflows can be found [here](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/).
```

Comment:
please apply this comment here as well: https://github.com/newrelic/terraform-provider-newrelic/pull/2685/files#r1630750618